docs/node-operators/get-started/setup-datanode.md

```diff
@@ -187,9 +244,11 @@ If you're using network history to get the current state to start up your data n
 2. Initialise the data node config files
 
 ```
-vega datanode init --home=$YOUR_DATANODE_HOME_PATH "vega-mainnet-0011" --archive
+vega datanode init --home=$YOUR_DATANODE_HOME_PATH "vega-mainnet-0011"
 ```
 
+By default the data node will retain all historic data generated by the chain. This is the expected behaviour for public data nodes. Find out more about data retention see the section on [data node retention policies](#data-node-retention-profiles).
```

Comment:
I don't think it needs to be here 2x, this second instance can go unless you have a specific reason for repeating it :)